packages/framework-provider-local/src/library/read-model-adapter.ts

```diff
@@ -59,6 +60,12 @@ export async function storeReadModel(
     throw e
   }
   logger.debug('Read model stored')
+  try {
+    await graphQLService.handleNotificationSubscription([{ typeName: readModelName, value: readModel }])
+    logger.debug('Read model change notified')
+  } catch (e) {
+    logger.error('Error notifying subscription', readModel)
+  }
```

Comment:
Not for this PR, but it makes a lot of sense to add a SubscriptionNotificationError to the Global error handler, as we have with the reductions, projections, etc

Reply:
Good ideaw